fastchat/conversation.py

```diff
@@ -1190,6 +1190,32 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# Yi template
+# reference: https://huggingface.co/01-ai/Yi-34B
+register_conv_template(
+    Conversation(
+        name = "yi-34b",
+        system_template="<|system|>\n{system_message}",
+        roles=("<|user|>", "<|assistant|>"),
+        sep_style=SeparatorStyle.ADD_COLON_TWO,
+        sep="\n",
+        sep2="\n\n",
+    )
+)
```

Comment:
Where do you find this template? I thought Yi is a base model without chat fine-tuned?

Reply:
oh you're right, I tried to follow the other examples
is there a way if there's no chat template?